source/collections/Queue.ooc

```diff
@@ -89,4 +89,7 @@ VectorQueue: class <T> extends Queue<T> {
 		this _capacity = newCapacity
 	}
 	toString: func -> String { "Queue: count=" << this _count toString() >> " capacity=" & this _capacity toString() }
+	operator [] (index: Int) -> T {
+		this _backend[this _tail - index % this _capacity]
```

Comment:
Not sure I understand the math here. Why are you indexing from the tail? If I want queue[0], do I want the tail or the head? And what happens when `index > tail`?